acbu_oracle/src/lib.rs

```
@@ -792,6 +792,10 @@ impl OracleContract {
             .unwrap()
     }
 
+    pub fn get_rate_decimals(env: Env) -> u32 {
```

Clippy (1.96.0)

[warning] unused variable

(warning)